server-code/Projects.js

```diff
@@ -82,3 +93,63 @@ function getProjectsObj(wbsNum) {
     };
     return project;
 }
+
+/** 
+ * Goes through spreadsheet data to get the WBS#s for the work packages associated with the given project and
+ * then returns a table containing that information.
+ * 
+ * @param {String} project – The Work Breakdown Structure # to find data/build a project object for
+ * @return {String[][]} – A 2-D Array representing the table of WBS#s
+ */
+function getProjectWorkPackagesTable(project) {
+    var data = getSheetInfo('mainSheetID', 'Work Packages', 'data');
+    var headers = data[0];
+    var wbsColIdx = findIdx("WBS #", headers);
+    var projWBSNum = project.wbsNum;
+    var projectWorkPackagesTable = [["Project WPs"]];
+    for (var rowIdx = 1; rowIdx < data.length; rowIdx++) {
+        var wpWBSNum = data[rowIdx][wbsColIdx];
+        if ((wpWBSNum.slice(0, -2) == projWBSNum.slice(0, -2))) {
+            projectWorkPackagesTable.push([wpWBSNum]);
+        }
+        // in this case, you have already discovered the associated WPs, so no unnecessary iterations
+        // compare to len 1 because you'll always have the header list
+        else if (projectWorkPackagesTable.length > 1) {
+            break;
+        }
```

Comment:
Do you think there is a better way to do this?

Reply:
What is this for? Also pay attention that the following are all valid WBS #s:

- 1.1.1
- 1.2.0
- 1.15.4
- 1.12.13
- 2.8.8

Reply:
Trying to find whether a wp is a child of a project?
Commit aab3cebd185b2f38f5d2a1fb23b8529ad93ca300 fixes the bug you referred to